src/diagnostics/LogPointsManager.ts

```diff
@@ -195,9 +209,11 @@ export class LogPointsManager extends vscode.Disposable {
 
         let logpoint = debugSessionManager.getLogpointAtLocation(uri, line);
         if (logpoint) {
-            debugSessionManager.removeLogpoint(logpoint);
-            debugSessionManager.unregisterLogpoint(uri, logpoint);
-            this._outputChannel.appendLine(`Removed logpoint at line ${logpoint.line} in ${params.path}`);
+            const succeed = await debugSessionManager.removeLogpoint(logpoint);
+            if (succeed) {
+                debugSessionManager.unregisterLogpoint(uri, logpoint);
+                this._outputChannel.appendLine(`Removed logpoint at line ${logpoint.line} in ${params.path}`);
+            }
```

Comment:
What happens if we don't succeed?

Reply:
We will not unregister the logpoint. The glyph will still show. And the error message is emitted from within the `removeLogpoint()` method.